pkg/operation/botanist/component/istio/charts/istio/istio-ingress/templates/networkpolicies.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Instead of adding this chart template, should we rather use Go structs in the `ingress_gateway.go` and add the `NetworkPolicy`s from there? IIUC, the Helm chart only exists for syncing/updating it conveniently from upstream istio. However, these `NetworkPolicy`s are purely related to Gardener/custom, so potentially we can follow the idiomatic way of how Gardener deploys resources (Go structs instead of Helm templates)?
cc @ScheererJ @DockToFuture

Reply:
Agreed.